src/dc_receive_imf.rs

```diff
@@ -1515,8 +1524,20 @@ async fn create_or_lookup_mailinglist(
         return (chat_id, blocked);
     }
 
+    if listid.ends_with(".list-id.mcsv.net") {
```

Comment:
A comment that it is a hack for Mailchimp which puts nonsense into human-readable part of the List-ID would be nice here, it's hard to find out what `mcsv.net` is otherwise.

Reply:
+1, i forgot about that, will add a commit